### PR TITLE
New package: Envoy49.go-spotify-cli version 1.0.41, ARM64

### PR DESCRIPTION
**Pull request by Envoy49**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/132469)

### CODE REVIEW

File: manifests/e/Envoy49/go-spotify-cli/1.0.41/Envoy49.go-spotify-cli.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Envoy49.go-spotify-cli
+PackageVersion: 1.0.41
+InstallerType: exe
+InstallerSwitches:
+  Silent: /silent
+  SilentWithProgress: /silent-progress
```
Comment (Trenly) [inline manifests/e/Envoy49/go-spotify-cli/1.0.41/Envoy49.go-spotify-cli.installer.yaml:9]:
```suggestion
InstallerType: portable
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Envoy49** [comment]
@Trenly looks like installation validation completed here with warning

https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=527623&view=logs&j=68997d86-8ece-5ae8-3ac5-ccdbfa458731&t=b79910ce-dca5-5e27-ad7f-bd285eecb5a2

**Comment by Trenly** [comment]
> @Trenly looks like installation validation completed here with warning
> 
> https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=527623&view=logs&j=68997d86-8ece-5ae8-3ac5-ccdbfa458731&t=b79910ce-dca5-5e27-ad7f-bd285eecb5a2

Thats about what I expected - since the validation pipelines don’t have an arm64 runner to install packages in. I'm hoping that the other PR also passes validation with just a warning, since then they could be combined into a single manifest (one entry for x64 systems and one for arm64 systems)

**Comment by Envoy49** [comment]
> > @Trenly looks like installation validation completed here with warning
> > https://dev.azure.com/ms/winget-pkgs/_build/results?buildId=527623&view=logs&j=68997d86-8ece-5ae8-3ac5-ccdbfa458731&t=b79910ce-dca5-5e27-ad7f-bd285eecb5a2
> 
> Thats about what I expected - since the validation pipelines don’t have an arm64 runner to install packages in. I'm hoping that the other PR also passes validation with just a warning, since then they could be combined into a single manifest (one entry for x64 systems and one for arm64 systems)

So with which PR are we going forward? Should I close this PR and wait concentrate on another? Or we need both?

**Comment by Trenly** [comment]
Lets wait until we know how the pipelines handle the other one and then I'll provide some guidance; Don't want to accidentally steer you in the wrong direction

**Comment by Envoy49** [comment]
@Trenly I think that I know why the X64 is stuck now.
But the ARM64 passed with warning.
So the .exe link which I provided for ARM64 is built using ARM64 architecture. 
`GOOS=windows GOARCH=amd64 go build -o go-spotify-cli.exe`

I believe that in order to pas x64 I need to provide another link which is build with x64 architecture.

**Comment by Envoy49** [comment]
> @Trenly I think that I know why the X64 is stuck now. Last time it was stuck 2 hours like this. But the ARM64 passed with warning. So the .exe link which I provided for ARM64 is built using ARM64 architecture. `GOOS=windows GOARCH=amd64 go build -o go-spotify-cli.exe`
> 
> Same link is used for X64 build but the link provided built using ARM64. I believe that in order to pass x64 I need to provide another link which is built with x64 architecture. Do we really need both? Isn't arm64 enough for windows 10 and 11?
> 
> EDIT: just googled it and seems like `GOOS=windows GOARCH=amd64 go build -o go-spotify-cli.exe` is the only option. So no issue with the link

ahh nevermind, another one passed without warnings, what is our next steps?

**Comment by Trenly** [comment]
> > @Trenly I think that I know why the X64 is stuck now. Last time it was stuck 2 hours like this. But the ARM64 passed with warning. So the .exe link which I provided for ARM64 is built using ARM64 architecture. `GOOS=windows GOARCH=amd64 go build -o go-spotify-cli.exe`
> > Same link is used for X64 build but the link provided built using ARM64. I believe that in order to pass x64 I need to provide another link which is built with x64 architecture. Do we really need both? Isn't arm64 enough for windows 10 and 11?
> > EDIT: just googled it and seems like `GOOS=windows GOARCH=amd64 go build -o go-spotify-cli.exe` is the only option. So no issue with the link
> 
> ahh nevermind, another one passed without warnings, what is our next steps?

I'll approve the x64 one so that it can be merged in; then you should be able to add the arm64 version as an installer node in the manifest, and it should pass without warning since the x64 will already be present.

Basically - 
1) Wait for other PR to merge
2) Rebase this PR on master or pull master into this branch, resolving merge conflicts 
3) Push changes

**Comment by Envoy49** [comment]
Sorry @Trenly I need to rebase properly

**Comment by Envoy49** [comment]
@Trenly all good to build now

**Review by Trenly — APPROVED** [review]

**Comment by ClintonOffor** [comment]
Ok

**Review by ClintonOffor — CHANGES_REQUESTED** [review]
Ok

**Pull request merged**